.github/workflows/run-terraform.yml

```diff
@@ -145,7 +145,9 @@ jobs:
     env:
       # makes some minor adjustments to Terraforms output to de-emphasize specific commands to run
       TF_IN_AUTOMATION: true
-      TF_WORKSPACE: ${{ inputs.terraform_workspace }}
+      WORKSPACE: ${{ inputs.terraform_workspace }}
+      TF_BACKEND_CONFIG: ${{ inputs.terraform_backend_config }}
+      TF_OPTIONS: ${{ inputs.terraform_options }}
```

Comment:
Obs, sender du disse via env vil begge slutte å virke for flere enn 1 opsjon :) Så dette er en breaking change som vi må ta senere

Reply:
Fungerer det for flere enn én opsjon nå? Skjønner ikke helt hvordan dette gjør en endring foruten å unngå RCE

Reply:
Om man skal unngå RCE her må API-et skrives om til f.eks. `terraform_option_1`, `terraform_option_2` og så videre. Det blir da en breaking change.

Reply:
aha, skjønner :)

skal man release denne og #17 sammen som 3.0.0, da?

Reply:
Ja, når man først ser på å endre API-et bør man ta med alt som er breaking samtidig, det er nok lurt